File: QingLian-app-root/src/main/java/com/yychainsaw/pojo/vo/TokenVO.java
package com.yychainsaw.pojo.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TokenVO {
    private String token;
    private UserVO userInfo;
}

File: qinglian-app-android/app/src/main/java/com/yychainsaw/qinglianapp/data/model/vo/TokenVO.kt
package com.yychainsaw.qinglianapp.data.model.vo

import com.google.gson.annotations.SerializedName

// 对应接口文档中的 TokenVO
data class TokenVO(
    val token: String,

    @SerializedName("userId", alternate = ["user_id", "id"])
    val userId: String,
    val username: String,
    val nickname: String?,
    @SerializedName("avatar", alternate = ["avatar_url", "avatarUrl", "headImg"])
    val avatar: String?
)
